js/modules/k6/html/html.go

```diff
@@ -71,3 +71,60 @@ func (s Selection) Attr(name string, def ...goja.Value) goja.Value {
 	}
 	return s.rt.ToValue(val)
 }
+
+func (s Selection) Html() goja.Value {
+	val, err := s.sel.Html()
+	if err != nil {
+		return goja.Undefined()
+	}
+	return s.rt.ToValue(val)
+}
+
+func optionVal(s *goquery.Selection) string {
+	val, exists := s.Attr("value")
+
+	if exists {
```

Comment:
imo the code is easier to read with no newline above this, same for eg.

```go
thing, err := Fn()
if err != nil {
  // …
}
```

Makes it immediately obvious which lines are related to each other.